### DISCUSSION TITLE
span `system` and `system overhead`

### DISCUSSION THREAD

**Discussion by anhnguyensgu** [bevyengine/bevy#6794] [Q&A] [answered]
After taking a look at `executor_parallel.rs `, I saw 2 spans `system` and `system overhead`. However, in `executor.rs`, we only have 1 span named `system`. I am just curious about why we need to use 2 different spans in `executor_parallel.rs `. In my case, I am writing a plugin to calculate the execution time of each system running in executors (`executor_parallel.rs `), which span should I use?

**Comment by james7132** [comment, marked as answer]
`system` is the actual time spent running the system itself. `system_overhead` is dedicated to the extra bookkeeping that is used to keep the scheduler running for each system.
